src/applications/facility-locator/tests/e2e/facilitySearch.cypress.spec.js

```diff
@@ -318,4 +318,43 @@ describe('Facility VA search', () => {
     cy.injectAxe();
     cy.axeCheck();
   });
+
+  it('does not trigger repeat API requests when submit is clicked multiple times', () => {
+    cy.intercept('GET', '/geocoding/**/*', mockGeocodingData);
+
+    cy.visit('/find-locations');
+
+    cy.injectAxe();
+    cy.axeCheck();
+
+    cy.verifyOptions();
+
+    cy.get('#street-city-state-zip').type('Austin, TX');
+    cy.get('#facility-type-dropdown')
+      .shadow()
+      .find('select')
+      .select('VA health');
+    cy.get('#service-type-dropdown').select('Primary care');
+
+    // Click the search button multiple times
+    cy.get('#facility-search').click({ waitForAnimations: true });
+    cy.get('#facility-search').click({ waitForAnimations: true });
+    cy.get('#facility-search').click({ waitForAnimations: true });
+    cy.get('#facility-search').click({ waitForAnimations: true });
```

Comment:
I think between the clicks here, you should add touching the other fields (not changing them...) to see if that does anything to the redux state that would allow another search.